src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/index.jsx

```diff
@@ -216,35 +249,57 @@ class GlobalSelectionHeader extends React.Component {
     return new Set(this.props.organization.features).has('global-views');
   };
 
-  didQueryChange = (prevProps, nextProps) => {
+  /**
+   * Identifies if query string has changed (with query params that this component cares about)
+   *
+   *
+   * @return {String[]} Returns `false` if did not change, otherwise return an array of params that have changed
+   */
+  changedQueryKeys = (prevProps, nextProps) => {
```

Comment:
This could probably stay as `didQueryChange` -> it seems like we still only need to know whether there were any changed keys or not, not what the changes are? Function description on above lines might be slightly out of date now?